kbi/0024/index.rst

```diff
@@ -0,0 +1,263 @@
+.. index::
+   single: RIA; archive
+.. highlight:: console
+
+KBI0024: Create and update a 7-Zip archive in a RIA store
+=========================================================
+
+:authors: Laura Waite <laura@waite.eu>, Michael Hanke <michael.hanke@gmail.com>
+:discussion: <link>
+:keywords: RIA, archive, 7-Zip, inodes
+:software-versions: datalad_0.18.3
+
+When working with `RIA stores`_, it is possible to compress the
+``annex/objects`` into `7-zip`_ archives. With this approach, the entire
+annex object store can be put into an archive and remain fully accessible while
+minimizing inodes, regardless of file number and size. This is beneficial
```

Comment:
```suggestion
minimizing `inodes`_, regardless of file number and size. This is beneficial
```

Maybe also add a link to the term definition - to me this used to be a foreign sounding word. Not necessary though, we use many other technical terms across the KBI, and it's a quick search away.

If this suggestion is accepted, a link should go at the bottom of the page.

Reply:
Good idea.